api/src/controllers/utils.ts

```diff
@@ -124,13 +122,39 @@ router.post(
 		const busboy = Busboy({ headers });
 
 		busboy.on('file', async (_fieldname, fileStream, { mimeType }) => {
-			try {
-				await service.import(req.params['collection']!, mimeType, fileStream);
-			} catch (err: any) {
-				return next(err);
-			}
-
-			return res.status(200).end();
+			const { tmp } = await import('@directus/utils/node');
+
+			const tmpFile = await tmp.createFile().catch(() => null);
+
+			if (!tmpFile) throw new Error('It was not possible to create a temporary file');
+
+			fileStream.pipe(fs.createWriteStream(tmpFile.path));
```

Comment:
We can't pass a stream to node worker.
Therefore we need to write a temporary file and pass the path to the worker